www/source/stylesheets/_global.scss

```diff
@@ -399,9 +399,17 @@ code {
     color: $white;
 }
 
+h2
+  a::before {
+    display: block;
+    content: "";
+    height: 220px;
+    margin: -220px 0 0;
+  }
+
```

Comment:
Adding brackets around the `h2`'s contents would keep it consistent with the rest of the document. Adding a `visibility: hidden;` and `pointer-events: none;` could help indicate that this element is solely for layout purposes. Adding a comment about why this block is needed would also be helpful.

Reply:
Thanks @davidMcneil! I tried using a height variable in the new code.